src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -51,14 +62,14 @@ export const styles = StyleSheet.create({
     alignSelf: 'center',
     bottom: 5
   },
-  countUser: {
-    fontFamily: fonts.inter[600],
+  countUser: (from) => ({
     fontSize: normalizeFontSize(14),
     lineHeight: normalizeFontSize(16.94),
-    color: colors.holytosca,
-    marginLeft: 20,
-    marginBottom: 4
-  },
+    color: from === SIGNED ? colors.darkBlue : colors.holytosca,
+    marginLeft: dimen.normalizeDimen(20),
+    marginBottom: dimen.normalizeDimen(4),
+    fontWeight: 'bold'
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `countUser` style now depends on the `from` parameter. This seems to be a good change as it allows for dynamic styling based on the context of the user. However, ensure that the `from` parameter always has a valid value (either `SIGNED` or any other expected value) to avoid unexpected behavior.



<!-- This is an auto-generated comment by OSS CodeRabbit -->